python/ray/tests/test_environ.py

```diff
@@ -31,6 +33,34 @@ def get_os_environ(self):
     assert len(actor_os_environ) > 0
 
 
+def test_update_envs():
+    with unittest.mock.patch.dict(os.environ):
+        env_vars = {
+            "PATH": "/test/lib/path:${PATH}",
+            "LD_LIBRARY_PATH": "/test/path1:${LD_LIBRARY_PATH}:./test/path2",
+            "DYLD_LIBRARY_PATH": "${DYLD_LIBRARY_PATH}:/test/path",
+            "LD_PRELOAD": "",
+        }
+        old_path = os.environ["PATH"]
+        os.environ["LD_LIBRARY_PATH"] = "./"
+        os.environ["DYLD_LIBRARY_PATH"] = "/lib64"
+        os.environ["LD_PRELOAD"] = "/lib:/usr/local/lib"
+        update_envs(env_vars)
+        assert os.environ["PATH"] == "/test/lib/path:" + old_path
+        assert os.environ["LD_LIBRARY_PATH"] == "/test/path1:./:./test/path2"
+        assert os.environ["DYLD_LIBRARY_PATH"] == "/lib64:/test/path"
+        assert os.environ["LD_PRELOAD"] == env_vars["LD_PRELOAD"]
+
+        # Test the empty string scenario
```

Comment:
1. The expected result is actually very simple, that is, the replaced string, and there is no need to explain too much.
2. I think if it is explained, it will lead to too many codes